packages/strapi-plugin-documentation/config/settings.json

```diff
@@ -17,7 +17,13 @@
   },
   "x-strapi-config": {
     "path": "/documentation",
-    "showGeneratedFiles": true
+    "showGeneratedFiles": true,
+    "pluginsForWhichToGenerateDoc": [
+      "email",
+      "upload",
+      "users-permissions"
+    ],
```

Comment:
In order to generate doc for all plugins by default, I would remove `pluginsForWhichToGenerateDoc` from the settings